tutorial/developing_packages/editable_packages/run_example.py

```diff
@@ -0,0 +1,53 @@
+import platform 
+
+from test.examples_tools import run, chdir, replace_in_file
+
+print("- Editable packages -")
+
+run("conan editable add say say/1.0")
+
+with chdir("say"):
+    if platform.system() == "Windows":
+        run("conan install . -s build_type=Release")
+        run("conan install . -s build_type=Debug")
+        run("cmake --preset default")
```

Comment:
Should we use the presets (cmake 3.23) in our docs beyond the explicit presets documentation? I know it is convenient, but for lower CMake versions it could be a bit confusing. I am fine with it, but maybe should be explicitly said in the docs.

It is also foreseeable that users will want to do presets for multiple projects, each one with each own presets, and aggregate them, but they conflict by preset name. Maybe we need to add the ``self.name`` to build_folder_vars finally, something to investigate (not for this docs)

Reply:
Yes, I had the same question with the presets, I was thinking that maybe we could use presets by default and add a note in the docs that if you don't want to use presets you should type something like this:

```
cmake <path> -G "Unix Makefiles" -DCMAKE_TOOLCHAIN_FILE=./build/Release/generators/conan_toolchain.cmake -DCMAKE_POLICY_DEFAULT_CMP0091=NEW -DCMAKE_BUILD_TYPE=Release
```

I'm still working in giving the structure, but definitely have that in mind